### PR TITLE
workload yaml change

### PR DESCRIPTION
**Pull request by akshaym99**
Signed-off-by: akshaym99 <akshay.s.mankar@seagate.com>

## Problem Statement

-   Workload yaml file change for fixed size object operations.

## Design

-   For Bug, Describe the fix here.
-   For new test addition post the link for design

## Coding

   Checklist for Author

-   [ ]  Coding conventions are followed and code is formatted (e.g. using pycharm in-built formatter)

## Testing

  Checklist for Author

-   [ ]  New/Affected tests are executed
-   [ ]  Attach test execution logs

## Review 

  Checklist for Author

-   [ ]  JIRA number/GitHub Issue added to PR
-   [ ]  PR is self reviewed
-   [ ]  Jira and state/status is updated and JIRA is updated with PR link
-   [ ]  Check if the description is clear and explained

## Documentation

  Checklist for Author

-   [ ]  Changes done to ReadMe

### CODE REVIEW

**Comment by akshaym99** [comment]
@VikasKumarC, @DhobaleVishal Please review/approve.

File: src/commons/workload_mapping.py [hunk 1]
```diff
@@ -39,4 +40,5 @@
     'object_range_read': [object_operations.TestS3Object, 'execute_object_workload'],
     'copy_object_fix_size': [copy_object.TestS3CopyObjects, 'execute_copy_object_workload'],
     'mix_object_ops': [mix_object_operations.TestMixObjectOps, 'execute_mix_object_workload']
```
Comment (DhobaleVishal) [inline src/commons/workload_mapping.py]: invalid syntax (F999). comma missing in 42.

File: src/commons/workload_mapping.py [hunk 2]
```diff
@@ -39,4 +40,5 @@
...
+    'object_fix_size_s3bench': [s3bench.S3bench , 'execute_s3bench_workload']
```
Comment (DhobaleVishal) [inline src/commons/workload_mapping.py]: Make sure changes are working fine by executing related workload.

**Review by DhobaleVishal — CHANGES_REQUESTED** [review]

File: src/commons/workload_mapping.py (hunk 1 above)
Comment (akshaym99) [reply src/commons/workload_mapping.py]: corrected now.

**Review by ketanarlulkar — APPROVED** [review]

File: src/commons/workload_mapping.py [hunk 3]
```diff
@@ -38,5 +39,6 @@
     'object_fix_size': [object_operations.TestS3Object, 'execute_object_workload'],
     'object_range_read': [object_operations.TestS3Object, 'execute_object_workload'],
     'copy_object_fix_size': [copy_object.TestS3CopyObjects, 'execute_copy_object_workload'],
-    'mix_object_ops': [mix_object_operations.TestMixObjectOps, 'execute_mix_object_workload']
+    'mix_object_ops': [mix_object_operations.TestMixObjectOps, 'execute_mix_object_workload'],
+    'object_fix_size_s3bench': [s3bench.S3bench, 'execute_s3bench_workload']
```
Comment (DhobaleVishal) [inline src/commons/workload_mapping.py]: If this is tested and working then keep here otherwise remove it.

**Review by DhobaleVishal — CHANGES_REQUESTED** [review]

**Review by DhobaleVishal — APPROVED** [review]

**Review by VikasKumarC — APPROVED** [review]

**Pull request merged**